pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -165,14 +165,23 @@ def render(self, task) -> Text:
 class RichProgressBarTheme:
     """Styles to associate to different base components.
 
+    Args:
+        text_color: Colour for progress bar titles.
```

Comment:
@kaushikb11 I think this can be worded better, but I could not think of anything else 😅. Any suggestions?

Reply:
@Raahul-Singh I did comments regarding it!